Game/Source/GDKTestGyms/Private/TestGymsReplicationGraph.cpp

```diff
@@ -0,0 +1,754 @@
+// Copyright 1998-2019 Epic Games, Inc. All Rights Reserved.
+
+/**
+*	
+*/
+
+#include "TestGymsReplicationGraph.h"
+
+#include "Net/UnrealNetwork.h"
+#include "Engine/LevelStreaming.h"
+#include "EngineUtils.h"
+#include "CoreGlobals.h"
+
+#include "GameFramework/Character.h"
+#include "GameFramework/GameModeBase.h"
+#include "GameFramework/GameState.h"
+#include "GameFramework/PlayerState.h"
+#include "GameFramework/Pawn.h"
+#include "Engine/LevelScriptActor.h"
+
+DEFINE_LOG_CATEGORY( LogTestGymsReplicationGraph );
+
+int32 CVar_TestGymsRepGraph_DisableSpatialRebuilds = 1;
+static FAutoConsoleVariableRef CVarTestGymsRepDisableSpatialRebuilds(TEXT("TestGymsRepGraph.DisableSpatialRebuilds"), CVar_TestGymsRepGraph_DisableSpatialRebuilds, TEXT(""), ECVF_Default );
+
+// ----------------------------------------------------------------------------------------------------------
+
+
+UTestGymsReplicationGraph::UTestGymsReplicationGraph()
+{
+	static ConstructorHelpers::FClassFinder<AActor> ReplicatedBP(TEXT("/Game/Actors/ReplicatedActor"));
+	if (ReplicatedBP.Class != nullptr)
+	{
+		ReplicatedBPClass = ReplicatedBP.Class;
+	}
+}
+
+void InitClassReplicationInfo(FClassReplicationInfo& Info, UClass* Class, bool bSpatialize, float ServerMaxTickRate)
```

Comment:
@mironec @improbable-danny In this file, worth looking at the bits where functionality differs according to the `UsesSpatialNetworking` flag